src/core/biblio-db.js

```diff
@@ -172,9 +192,18 @@ export const biblioDB = {
       throw new TypeError("Invalid alias object.");
     }
     const db = await this.ready;
-    const isInDB = await this.has(type, details.id);
-    const store = db.transaction(type, "readwrite").store;
+    let isInDB = await this.has(type, details.id);
     // update or add, depending of already having it in db
+    // or if it's expired
+    if (isInDB) {
+      const entry = await this.get(type, details.id);
+      if (entry?.expires < Date.now()) {
+        const { store } = db.transaction(type, "readwrite");
```

Comment:
Aren't we suppose to `await tx.done`? Maybe `db.objectStore` would be nicer?

Reply:
> Aren't we suppose to await tx.done? 

I don't think it has a .done()? is that something Jake added to the library? 

> Maybe db.objectStore would be nicer?

sorry, not sure what you are suggesting here... happy to change it if there is something better.